docs/source/api_reference/models.rst

```diff
@@ -538,6 +538,13 @@ WeaveModel
   :members:
 
 
+ProgressiveMultitaskModel
+-------------------------
+
+.. autoclass:: deepchem.models.torch_models.ProgressiveMultitaskModel
```

Comment:
@gauthamk02 Can you please fix the indentation here?

Reply:
Fixed